ztools/x509/pkix/json.go

```diff
@@ -175,20 +175,21 @@ func (o *OtherName) MarshalJSON() ([]byte, error) {
 	return json.Marshal(oName)
 }
 
-func (o *OtherName) UnmarshalJSON(b []byte) error {
+func (o *OtherName) UnmarshalJSON(b []byte) (err error) {
 	var oName jsonOtherName
 
-	if err := json.Unmarshal(b, &oName); err != nil {
-		return err
+	if err = json.Unmarshal(b, &oName); err != nil {
+		return
 	}
 
 	arcs := strings.Split(oName.Id, ".")
 	oid := make(asn1.ObjectIdentifier, len(arcs))
 
 	for i, s := range arcs {
-		tmp, err := strconv.ParseInt(s, 10, 32)
+		var tmp int64
```

Comment:
It was changed because it won't compile otherwise. I could change it to `tmp, e := strconv.ParseInt(s, 10, 32)` if you like. That should work.